test/bigtest/tests/patron-blocks/settings-limits-form-test.js

```diff
@@ -91,6 +103,17 @@ describe('Patron blocks limits form', () => {
         expect(SettingsLimitsForm.errorMessage.text).to.equal(feeFineValidationMessage);
       });
     });
+
+    describe('set float limit value for fee fine limit', () => {
+      beforeEach(async function () {
+        await SettingsLimitsForm.limitField(5)
+          .fillAndBlur(12.5);
+      });
+
+      it('should show error message', () => {
+        expect(SettingsLimitsForm.errorMessage.isPresent).to.be.false;
+      });
```

Comment:
"should show error message ... expect errorMessage is present ... to be false"

This doesn't add up to me.

Reply:
@zburke, do you mean will be appropriate to rewrite it to "should not show error message"?

Reply:
The test title is effectively "error message is present" but the test passes when `errorMessage.isPresent` is `false`. One of those conditions needs to be flipped.

Reply:
I rewrote test, looks like make more sence to show that we can save float value for current limit, not just show that there are not error message